packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -134,24 +138,25 @@ export const OptionEditor: React.FunctionComponent<OptionEditorProps> = ({
   }, []);
 
   return (
-    <Editor
-      variant={EditorVariant.Shell}
+    <div
       className={cx(
         editorStyles,
         focusRingProps.className,
         hasError && editorWithErrorStyles
       )}
-      theme="mongodb-query"
-      text={value}
-      onChangeText={(value) => onChange(value)}
-      id={id}
-      options={editorSettings}
-      completer={completer.current}
-      placeholder={placeholder}
-      fontSize={12}
-      scrollMargin={[6, 6, 0, 0]}
-      onLoad={onLoadEditor}
-    />
+    >
+      <Editor
+        variant="Shell"
+        className="inline-editor"
```

Comment:
Maybe we can make it into a special `inline` prop on the component that will also disable the gutters and adds this class name instead of passing it here directly. This is the only case where we need it now though, so I didn't want to over abstract this for now

Reply:
If I understand it right I having an `inline` prop is better even if we have this one instance alone, there is not need to break encapsulation and exposing back the internals of the editor, we have 2 types of editors after all, not coding it explicitly would create the same "concept", just in a less obvious way.

Apart from that, it seems something went wrong with these fields between Crud and Aggregations:
<img width="1145" alt="Screenshot 2022-10-14 at 08 03 14" src="https://user-images.githubusercontent.com/334881/195774085-768f57b7-d929-41d1-b331-592b3914df1b.png">

<img width="836" alt="Screenshot 2022-10-14 at 08 03 19" src="https://user-images.githubusercontent.com/334881/195774103-1be3907f-4011-4758-92dc-84f6e0fb6e8d.png">

Reply:
Alright, I'll make it into a prop!

I think collation in aggregation is not using the editor, it's just leafygreen text input, so the styles and behavior are different because of that.

Reply:
Added `inline` prop in https://github.com/mongodb-js/compass/pull/3589/commits/ac4bdb92384ef8cdbfd7305eaf394d8e9de523d5